### DISCUSSION TITLE
Will there be an IOS edition?

### DISCUSSION THREAD

**Discussion by ivaquero** [namidaco/namida#144] [Q&A]
Hi, this is the best music player that I have ever used! 

Will there be an IOS edition? 

I'd like to have it on all my equips.

**Comment by MSOB7YY** [comment]
thank u so much for liking it! unfortunately the are no plans for supporting other platforms currently. maybe in future, if any plans appeared, then it will likely only be ios.

**Reply by idontveapc** [reply]
> it will likely only be ios

??
Doesn't it must be as Android (Material Design/You) or even iOS (HIG) (or any one of both) on *whatever* platform/device?

**Reply by MSOB7YY** [reply]
wdym? design wont change. 
what i mean is that ios is the most likely to get support, since many packages are not available on desktop + the dimensions will need to be re-adapted

**Reply by idontveapc** [reply]
Yup.
